geometry_msgs/Point pos
float32 hdg
